From ed974c7533623ef6256fc11cd29820178347ab52 Mon Sep 17 00:00:00 2001
From: Nic Costa <nic.costa@gmail.com>
Date: Thu, 14 May 2020 17:29:32 +0000
Subject: [PATCH] force port 443

the method to enable the port 443 override in the cloud client
config header doesn't work.  somehow the header isn't being
included by the file that checks the value.
---
 lib/mbed-cloud-client/mbed-client/source/m2minterfaceimpl.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/mbed-cloud-client/mbed-client/source/m2minterfaceimpl.cpp b/lib/mbed-cloud-client/mbed-client/source/m2minterfaceimpl.cpp
index b8998dc..54d18e0 100644
--- a/lib/mbed-cloud-client/mbed-client/source/m2minterfaceimpl.cpp
+++ b/lib/mbed-cloud-client/mbed-client/source/m2minterfaceimpl.cpp
@@ -946,6 +946,7 @@ void M2MInterfaceImpl::process_address(const String& server_address, String& ip_
     int colonFound = server_address.find_last_of(':'); //10
     if(colonFound != -1) {
         ip_address = server_address.substr(0,colonFound);
+#define MBED_CLOUD_CLIENT_CUSTOM_URI_PORT 443
 #ifndef MBED_CLOUD_CLIENT_CUSTOM_URI_PORT
         port = atoi(server_address.substr(colonFound+1,
                                          server_address.size()-ip_address.size()).c_str());
-- 
2.7.4

